From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 2 Jan 2018 18:34:30 +0000
Subject: [PATCH] Update cockpit tmpfile configuration

Update systemd tmpfile for creation of /etc directories required by
cockpit.
---
 src/systemd/cockpit-tempfiles.conf.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/systemd/cockpit-tempfiles.conf.in b/src/systemd/cockpit-tempfiles.conf.in
index 0dcddc2..5c30ce7 100644
--- a/src/systemd/cockpit-tempfiles.conf.in
+++ b/src/systemd/cockpit-tempfiles.conf.in
@@ -1,3 +1,6 @@
+d /etc/cockpit - root root - -
+d /etc/cockpit/machines.d - root root - -
+d /etc/cockpit/ws-certs.d - root root - -
 C /run/cockpit/inactive.motd 0640 root @admin_group@ - @datadir@/@PACKAGE@/motd/inactive.motd
 f /run/cockpit/active.motd   0640 root @admin_group@ -
 L+ /run/cockpit/motd - - - - inactive.motd
